impl/src/main/java/com/okta/sdk/impl/client/DefaultClientBuilder.java

```diff
@@ -471,6 +471,9 @@ private String getFileContent(InputStream privateKeyStream) {
     }
 
     private String readFromInputStream(InputStream inputStream) throws IOException {
+        if (inputStream == null) {
```

Comment:
Avoid throwing generic exceptions like `RuntimeException`. In this case, do an `Assert.notNull(..)` and that'll throw IllegalArgumentException.

Reply:
fixed